.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,73 @@
+name: Playwright Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
```

Comment:
```suggestion

permissions:
  pull-requests: write

```

Reply:
Based on [this discussion](https://github.com/orgs/community/discussions/26644) I believe if you add this permissions block, then It will prompt maintainers to approve workflows, and it will be able to do the test when we mash that button.